### PR TITLE
Update selenium version

### PR DESCRIPTION
**Pull request by emaks**
Fix for #89
Could you, please, make next release with this changes?

### CODE REVIEW

File: htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/HtmlElement.java [hunk 1]
```diff
@@ -252,4 +249,9 @@ public String getCssValue(String name) {
     public String toString() {
         return name;
     }
+
+    @Override
+    public <X> X getScreenshotAs(OutputType<X> outputType) throws WebDriverException {
```
Comment (aik099) [inline htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/HtmlElement.java]: PR title says that only Selenium version is updated, but this method is added as well.

Comment (emaks) [inline htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/HtmlElement.java]: according to https://github.com/SeleniumHQ/selenium/issues/1032

**Comment by emaks** [comment]
@artkoshelev are there any updates or problems with this changes?

File: htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/HtmlElement.java [hunk 2]
```diff
@@ -1,9 +1,6 @@
 package ru.yandex.qatools.htmlelements.element;
 
-import org.openqa.selenium.By;
-import org.openqa.selenium.Dimension;
-import org.openqa.selenium.Point;
-import org.openqa.selenium.WebElement;
+import org.openqa.selenium.*;
```
Comment (artkoshelev) [inline htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/HtmlElement.java]: please don't do wildcard imports

Comment (emaks) [inline htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/HtmlElement.java]: @artkoshelev  have been fixed

**Comment by artkoshelev** [comment]
@emaks great, squash commits now and we're done

**Comment by emaks** [comment]
@artkoshelev done

**Comment by artkoshelev** [comment]
@emaks merged, thank you!

**Comment by emaks** [comment]
@artkoshelev Could you make maven release with this changes?

**Comment by artkoshelev** [comment]
@emaks i think we can make release right after this PR https://github.com/yandex-qatools/htmlelements/pull/98

**Pull request merged**